src/EnergyPlus/DXCoils.cc

```diff
@@ -14474,6 +14474,29 @@ namespace DXCoils {
         return NodeNumber;
     }
 
+    int getCoilInNodeIndex(int const &CoilIndex, // coil index
```

Comment:
New mining function where only an index is known for getting inlet/outlet node number. I suspect that one or more of these sets can be removed.